jdbc-adapter/doc/supported_sql_dialects.md

```diff
@@ -0,0 +1,26 @@
+# Supported Dialects
```

Comment:
See my first comment about this new subpage

Reply:
Done.